contact.html

```diff
@@ -0,0 +1,152 @@
+                                aria-expanded="false">
+                                Dropdown
+                            </a> -->
+                            <!-- <ul class="dropdown-menu dropdown-menu-dark">
+                                <li><a class="dropdown-item" href="#"></a></li>
+                                <li><a class="dropdown-item" href="#"></a></li>
+                                <li>
+                                    <hr class="dropdown-divider">
+                                </li>
+                                <li><a class="dropdown-item" href="#">Something else here</a></li>
+                            </ul> -->
+                        </li>
+                    </ul>
+
+                </div>
+            </div>
+        </div>
+    </nav>
+
+
+
+    <div class="row">
+        <nav aria-label="breadcrumb">
+            <ol class="breadcrumb">
+                <li class="breadcrumb-item active" aria-current="page">Home</li>
+                <li class="breadcrumb-item active" aria-current="page">Contact</li>
+            </ol>
+        </nav>
+    </div>
+
+    <img class="mb-5 ms-5" src="assets/Main Text.png" alt="">
+
+    <div class="container" data-bs-theme="dark">
+
+        <div class="row">
+            <hr class="ligne">
+        </div>
+<p class="text-center">N'hésitez pas à m'écrire</p>
+
+        <div class="card ms-3" style="width: 18rem;">
+            <div class="card-body">
+                <form>
+                    <div class="form-group">
+                      <label for="exampleFormControlInput1">Votre mail</label>
+                      <input type="email" class="form-control" id="exampleFormControlInput1" placeholder="name@example.com">
+                    </div>
+                    <br>
+                    <div class="form-group">
+                        <label for="name" class="form-label">Nom</label>
+                        <input type="text" class="form-control" id="name" placeholder="Steve Jobs" required>
+                    </div>
+                    <br>
+                    <div class="form-group">
+                        <label for="exampleFormControlTextarea1">Écrivez moi</label>
+                        <textarea class="form-control" id="exampleFormControlTextarea1" rows="4"></textarea>
+                      </div>
+                      <br>
+                      <div class="col-12 mb-4">
+                        <button type="submit" class="btn btn-dark">Envoyer</button>
+                    </div>
+                     
+                      <p class="text-center">Au plaisir de vous rencontrer</p>
+                    </form>
+                    
+            </div>
+          </div>
+         
+
+                
+        
+
+       
+
+        <div class="row">
+            <hr class="ligne">
+        </div>
+
```

Comment:
Missing the Google Map.

Reply:
![image](https://user-images.githubusercontent.com/6362750/235273541-852f7b1f-758a-4885-bef7-66fb5eeaef32.png)